Automattic/wp-calypso
# Subscription management: Fix delivery frequency overflow

Closes https://github.com/Automattic/wp-calypso/issues/75463

## Proposed Changes

| Before | After |
|-------|------|
| <img src="https://user-images.githubusercontent.com/528287/230853141-2c90434b-52e5-4186-a7c4-e4bd11317aae.png" /> | <img width="449" alt="CleanShot 2023-04-10 at 13 00 30@2x" src="https://user-images.githubusercontent.com/528287/230890349-e9202ff2-224e-43b3-91f2-12f63f407d7a.png"> |

It changes `width: 305px;` to `min-width: 305px;`. I tested with `pt-BR` on an emulated iPhone XR, and the content still fits well within the viewport. 

We'll have to keep an eye on other translations.

## Testing Instructions

1. Apply this PR
2. Go to http://calypso.localhost:3000/subscriptions/sites?locale=pt-br
3. Click the three dots to get the Popover, the content should not overflow.

## Pre-merge Checklist
